src/shared_utilities/Regridder.jl

```diff
@@ -187,9 +208,6 @@ function hdwrite_regridfile_rll_to_cgll(
     topology = ClimaCore.Topologies.Topology2D(
         cpu_context,
         ClimaCore.Spaces.topology(space).mesh,
-        ClimaCore.Topologies.spacefillingcurve(
```

Comment:
Using `spacefillingcurve` produced a different topology, so `space_undistributed != space`, even in the undistributed case. Removing this call makes `space_undistributed == space` in the undistributed case

Reply:
is it worth leaving this as a comment in the code?

Reply:
good idea!